RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -637,6 +637,11 @@ void MuonIdProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
      }
    }
 
+   if (arbitrateTrackerMuons_){
+     fillArbitrationInfo( outputMuons.get() );
```

Comment:
in case arbitrateTrackerMuons_ is true, what will happen with the call at L721
```
 fillArbitrationInfo( outputMuons.get(), reco::Muon::TrackerMuon );
```
is the repeated call harmless or do we need an extra protection?